src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -54,7 +53,6 @@ proc newModule*(delegate: delegate_interface.AccessInterface, events: EventEmitt
   result.controller = controller.newController(result, events, sectionId, chatId, belongsToCommunity, contactService,
   communityService, chatService, messageService, mailserversService)
   result.moduleLoaded = false
-  result.initialMessagesLoaded = false
```

Comment:
This condition seems unnecessary, given that we are actually aiming to show the loading page animation until we have at least a single message to load. See PR description. It is also the root-cause for the animation lingering around for chats with no messages yet.

Reply:
A channel can be empty, meaning there may be no messages to load. In other words, `initialMessagesLoaded` can be set to `true` even if there are no messages present.